content/docs/azimuth/layer2.md

```diff
@@ -0,0 +1,387 @@
+Bridge, a ship may submit X transactions to Tlon's roller per Y period free of
+charge. Tlon's roller submits on a regular schedule: a submission occurs when a
+total of A layer 2 transactions have been submitted to it since the the previous
+submitted Ethereum transaction, or every Z time, whichever occurs first. (this
+should probably go in the operations manual)
+
+There are no security risks in utilizing an aggregator. The transactions you
+submit to it are signed with your private key, and so if an aggregator alters
+them the signature will no longer match and `naive.hoon` will reject it as an
+invalid transaction. The worst an aggregator can do is not submit your transaction.
+
+## Multi-keyfiles
+
+Maybe this section goes in operating manual?
+
+As part of the layer 2 upgrade, Tlon has expanded the role of
+[keyfiles](/docs/glossary/keyfile). One of our goals with layer 2 was to reduce
+the amount of friction experienced when getting onto Urbit. The enormous
+reduction in fees has made a new boot method which allows instantaneous sale of
+layer 2 planets or stars to be cost effective.
+
+The ideal situation would be for the end user to be able to buy or receive a
+planet and immediately boot it without having to wait for an aggregator to
+submit a transaction that spawns the planet. In order to bring about this
+circumstance, "multi-keyfiles" have been introduced.
+
+Multi-keyfiles are keyfiles used to boot an urbit for first time that contains
+more than one set of keys, the purpose of which is to initially utilize one set
+of keys on a temporary basis, and then the other set of keys soon after. This
+works as follows. A star owner prespawns a number of layer 2 planets ready to be
+sold at any time for which they posess the initial key. When a user acquires a
+planet from a star owner, the star owner immediately hands the initial keys to
+boot the planet to the buyer. The buyer then queues a transaction at a roller to
+rotate the keys to a new value only known by the buyer (this step is performed
+automatically by Bridge). The buyer then downloads a keyfile from Bridge
+containing both keys and uses that to boot their planet for the first time.
+After the transaction setting the new keys is submitted by the roller to
+Ethereum, the purchased planet will automatically switch to them once it reads
+the corresponding transaction from Ethereum.
+
+This process introduces a short period of time in which both the buyer and
+seller are in possession of the keys of a planet. A malicious seller could
+theoretically sell a planet to more than one party in this time period. However,
+they would quickly be found out as two identical ships on the network
+immediately creates problems, and the seller's reputation would be tarnished.
+Due to the expense or effort needed to acquire a star, this seems an unlikely
+scenario as the reward is much less than the cost. Nonetheless, buyers should
```

Comment:
Should mention that if you want to be sure you've received the planet, you should just wait for the batch to be sent and confirmed.